index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Michigan DevFest 2024</title>
+    <title>Michigan DevFest 2025</title>
```

Comment:
I think this branch is a little behind as this was done here - https://github.com/GDG-Detroit/devfest-website/commit/3e89502105473305380803a5751bb278a251cd34

Not that's it's going to conflict as it's the same change